features/feature-main/src/main/java/com/ownsnoretracker/feature_main/presentation/MainHomeScreen.kt

```diff
@@ -16,7 +34,44 @@ fun MainHomeScreen(
     Column(
         Modifier
             .fillMaxSize()
-            .background(mainTheme)) {
-        Text(text = "main")
+            .background(mainTheme)
+    ) {
+        var triggerAnimation by remember { mutableStateOf(false) }
+
+        Box(
+            contentAlignment = Alignment.TopCenter,
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(top = 50.dp)
+        ) {
+            PulsingImage(
+                triggerAnimation = triggerAnimation,
+                onAnimate = { triggerAnimation = it }
+            )
+        }
+
+        LaunchedEffect(triggerAnimation) {
+            if (triggerAnimation) {
+                delay(200)
+                triggerAnimation = false
+            }
+        }
     }
+}
+
+@Composable
+fun PulsingImage(
```

Comment:
🚫 **[ktlint]** <standard:function-naming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Function name should start with a lowercase letter (except factory methods) and use camel case